lib/kitchen/driver/azure_credentials.rb

```diff
@@ -50,16 +47,31 @@ def azure_options
 
       private
 
+      def logger
+        Kitchen.logger
+      end
+
+      def config_path
+        @config_path ||= File.expand_path(ENV["AZURE_CONFIG_FILE"] || CONFIG_PATH)
+      end
+
       def credentials
-        @credentials ||= {}
+        @credentials ||= begin
+          if File.file?(config_path)
+            IniFile.load(config_path)
+          else
+            warn "#{config_path} was not found or not accessible. Will attempt to use Managed Identity."
```

Comment:
Changed the wording here because you will only see this error message if the environment variables are not set _and_ the file is not found. This should probably be cleaned up and moved somewhere else.